includes/admin/settings/class-settings-general.php

```diff
@@ -3,592 +3,714 @@
+                            'title' => __('Access Control', 'give'),
+                            'type' => 'give_docs_link',
+                        ],
+                        [
+                            'id' => 'give_title_session_control_1',
+                            'type' => 'sectionend',
+                        ],
+                    ];
+                    break;
+
+                case 'currency-settings':
+                    $currency_position_before = __('Before - %s&#x200e;10', 'give');
+                    $currency_position_after = __('After - 10%s&#x200f;', 'give');
+
+                    $hasIntlExtension = class_exists(NumberFormatter::class);
+                    $formatDisabledAttribute = $hasIntlExtension || give_get_option('auto_format_currency') ? [] : [
+                        'disabled' => 'disabled',
+                        'title' => __('This option is disabled because the PHP Intl extension is not installed.', 'give'),
+                    ];
+
+                    $settings = [
+                        // Section 2: Currency
+                        [
+                            'type' => 'title',
+                            'id' => 'give_title_general_settings_2',
+                        ],
+                        [
+                            'name' => __('Currency Settings', 'give'),
+                            'desc' => '',
+                            'type' => 'give_title',
+                            'id' => 'give_title_general_settings_2',
+                        ],
+                        [
+                            'name' => __('Currency', 'give'),
+                            'desc' => __(
+                                'The donation currency. Note that some payment gateways have currency restrictions.',
+                                'give'
+                            ),
+                            'id' => 'currency',
+                            'class' => 'give-select-chosen',
+                            'type' => 'select',
+                            'options' => give_get_currencies(),
+                            'default' => 'USD',
+                            'attributes' => [
+                                'data-formatting-setting' => esc_js(wp_json_encode(give_get_currencies_list())),
+                            ],
+                        ],
+                        [
+                            'name' => __('Auto-format server currency amounts', 'give'),
+                            'desc' => sprintf(
+                            /* translators: %s: Link to the GiveWP docs. */
+                                __(
+                                    'Currencies will be automatically formatted based on the user\'s location. Requires the PHP intl-extension. Read more about this <a href="%s" target="_blank">in our docs.</a>',
+                                    'give'
+                                ),
```

Comment:
@Benunc Please give this some messaging love.

Reply:
I believe this is correct: we are not formatting based on user or donor location, but based on currency, right? An individual donor or admin user experience will not differ based on location. It'll format it for everybody based on the currency. 

```suggestion
                                    'If enabled, amounts are automatically formatted based on the selected currency. This option requires the intl extension for PHP. Read more about <a href="%s" target="_blank">this option and how GiveWP values internationalization in web applications.</a>',
                                    'give'
                                ),
```

Reply:
Also, we got rid of lots of checkboxes once upon a time in favor of radio "enabled/disabled" options. Not sure if we should continue that here, and make this enabled/disabled. My suggested message above leans that direction.